superset/assets/backendSync.json

```diff
@@ -1635,7 +1635,7 @@
       "freeForm": true,
       "label": "Number format",
       "renderTrigger": true,
-      "default": ".3s",
+      "default": "smart_number",
```

Comment:
If you use `SMART_NUMBER` (all caps), this will be already registered out of the box and you don't have to do anything in `superset/assets/src/setup/setupFormatters.js`

Reply:
My bad, I misunderstood. It's fixed now!